README.md

```diff
@@ -132,7 +132,221 @@ sqlite> .read Food_Locations.sql
 ```
 
 ### Visualize
-> **TODO:** Documentation of queries specific to use case 1 (U1)
+
+#### Results
+
+```sql
+sqlite> -- Results
+SELECT
+    Cleaned_Food_Licensee_Inspections."Results" AS "Results",
+    Cleaned_Food_Licensee_Inspections."Inspection ID" AS "Inspection ID",
+    (
+        SELECT count(*) 
+        FROM Cleaned_Food_Inspection_Violations
+        WHERE Cleaned_Food_Inspection_Violations."Inspection ID" = Cleaned_Food_Licensee_Inspections."Inspection ID"
+    ) AS "CNT(Violation ID)",
+    Cleaned_Food_Licensees."DBA Name" AS "DBA Name",
+    Cleaned_Food_Licensees."Address" AS "Address",
+    Cleaned_Food_Licensees."License #" AS "License #"
+FROM Cleaned_Food_Licensee_Inspections
+INNER JOIN Cleaned_Food_Licensees
+ON Cleaned_Food_Licensee_Inspections."License #" = Cleaned_Food_Licensees."License #"
+INNER JOIN Cleaned_Food_Locations
+ON Cleaned_Food_Licensees."Address" = Cleaned_Food_Locations."Address"
+    AND Cleaned_Food_Licensees."City" = Cleaned_Food_Locations."City"
+    AND Cleaned_Food_Licensees."State" = Cleaned_Food_Locations."State"
+    AND Cleaned_Food_Licensees."Zip" = Cleaned_Food_Locations."Zip"
+WHERE Cleaned_Food_Locations."Latitude" BETWEEN 41.9754 AND 42.0034
+    AND Cleaned_Food_Locations."Longitude" BETWEEN -87.8173 AND -87.7284
+    AND strftime('%Y-%m', Cleaned_Food_Licensee_Inspections."Inspection Date") = '2010-08'
```

Comment:
I am curious about these filters. Are they arbitrary values, or did you select them to make the difference in food location results more evident?

Reply:
PS: the selected item in each visualization corresponds to the last item in the table results.